app/models/concerns/template_tax.rb

```diff
@@ -0,0 +1,20 @@
+module TemplateTax
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def taxonomy_exportable
+      {
+        :organizations => method(:org_names),
+        :locations => method(:loc_names)
+      }
+    end
+
+    def org_names(template)
+      template.organizations.select { |org| User.current.admin ? true : User.current.my_organizations.include?(org) }.map(&:title)
```

Comment:
I think we shouldn't make a shortcut here and just call User.current.my_organizations that should always return all for admin. The admin exception should be kept and single place, and that's my_$taxonomy method. The same applies to loc_names.